canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -249,7 +250,7 @@ class FlexibleTable extends React.Component {
 		if (Array.isArray(this.props.data) && this.props.data.length < rows) {
 			newHeight = (rowHeight * this.props.data.length + headerHeight) + "rem";
 		} else if (rows > 0) {
-			newHeight = (rowHeight * rows + headerHeight) + "rem";
+			newHeight = (rowHeight * rows + headerHeight + (this.props.selectedEditRow ? REMOF2ROWS : 0)) + "rem";
```

Comment:
Instead of `REMOF2ROWS`, can you use the `rowHeight` and `headerHeight` defined here to indicate that we need to add a header and row to accommodate the multiselect table? Maybe a new const like:
```
const multiSelectTableHeight = rowHeight + headerHeight;
```
for readability